include/RAJA/util/resource.hpp

```diff
@@ -114,6 +129,11 @@ namespace RAJA
     using type = camp::resources::Sycl;
   };
 
+  template <bool Async, int num_threads>
+  struct get_resource<cuda_launch_t<Async, num_threads>>{
+    using type = camp::resources::Sycl;
```

Comment:
There's a mismatch here with the template parameter being `cuda_launch_t` and the alias being `camp::resources::Sycl`.

Reply:
ah copy paste error, thanks!